app/views/components/_content-metadata.html.erb

```diff
@@ -0,0 +1,18 @@
+<%
+  published ||= false
+  last_updated ||= false
+  from ||= false
+  metadata = [published, last_updated, from]
+%>
+<% if metadata.any? %>
+  <div class="app-c-content-metadata">
+    <div class="grid-row">
+      <div class="column-two-thirds">
+        <%= render 'components/published-dates', published: published, last_updated: last_updated %>
```

Comment:
This doesn't allow for a published dates component with [link_to_history](https://government-frontend.herokuapp.com/component-guide/published-dates/link_to_page_history) enabled